conf/routes

```diff
@@ -35,9 +35,9 @@ GET         /consignment/:consignmentId/transfer-complete
 GET         /consignment/:consignmentId/additional-metadata                                       controllers.AdditionalMetadataController.start(consignmentId: java.util.UUID)
 GET         /consignment/:consignmentId/additional-metadata/closure-status                        controllers.AdditionalMetadataClosureStatusController.getClosureStatusPage(consignmentId: java.util.UUID, fileIds: List[java.util.UUID])
 POST        /consignment/:consignmentId/additional-metadata/closure-status                        controllers.AdditionalMetadataClosureStatusController.submitClosureStatus(consignmentId: java.util.UUID, fileIds: List[java.util.UUID])
-GET         /consignment/:consignmentId/additional-metadata/closure/selected-summary              controllers.AdditionalMetadataSummaryController.getSelectedSummaryPage(consignmentId: java.util.UUID, fileIds: List[java.util.UUID])
-GET         /consignment/:consignmentId/additional-metadata/closure/confirm-delete-metadata       controllers.DeleteAdditionalMetadataController.confirmDeleteAdditionalMetadata(consignmentId: java.util.UUID, fileIds: List[java.util.UUID])
-GET         /consignment/:consignmentId/additional-metadata/closure/delete-metadata               controllers.DeleteAdditionalMetadataController.deleteAdditionalMetadata(consignmentId: java.util.UUID, fileIds: List[java.util.UUID])
+GET         /consignment/:consignmentId/additional-metadata/:metadataType/selected-summary        controllers.AdditionalMetadataSummaryController.getSelectedSummaryPage(consignmentId: java.util.UUID, metadataType: String, fileIds: List[java.util.UUID])
```

Comment:
Could the "metadataType" parameter go at the end of the URLs eg: `/consignment/:consignmentId/additional-metadata/selected-summary/:metadataType`

This would keep is consistent with the existing "files" URL.

Reply:
@TomJKing Shall I change it for `confirm-delete-metadata` and `delete-metadata` URLs too to make it consistent?